getting_started/scripting/gdscript/gdscript_advanced.rst

```diff
@@ -368,7 +368,7 @@ The range() function can take 3 arguments:
     range(b, n) # Will go from b to n-1
     range(b, n, s) # Will go from b to n-1, in steps of s
 
-Some examples:
+Some staticly typed programming language examples:
```

Comment:
"statically typed" seems 100 times more common than "staticly typed" according to Google (it's the first time I read the latter).

Reply:
@akien-mga : Thanks for catching another one! :grin: Definitely wrong! I was hurrying and not paying attention.